frontend/app/scripts/controllers/popups/waitListCtrl.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+angular.module('frontendApp')
+  .controller('WaitListCtrl', ['participants','$scope', function(participants, $scope) {
+
+    $scope.participants = participants.participants;
```

Comment:
Please use the same names as in the rest api specification doc. Please check it.